Cargo.toml

```
@@ -85,7 +85,7 @@ libp2p-floodsub = { version = "0.44.0", path = "protocols/floodsub" }
 libp2p-gossipsub = { version = "0.46.1", path = "protocols/gossipsub" }
 libp2p-identify = { version = "0.44.2", path = "protocols/identify" }
 libp2p-identity = { version = "0.2.8" }
-libp2p-kad = { version = "0.45.4", path = "protocols/kad" }
+libp2p-kad = { version = "0.45.5", path = "protocols/kad" }
```

```suggestion
libp2p-kad = { version = "0.45.4", path = "protocols/kad" }
```

`0.45.4` isn't released yet, we can pack this into the same release.

---

Fixed!